src/diagnostic-channel-publishers/package.json

```diff
@@ -32,7 +34,7 @@
     "redis": "^2.7.1",
     "rimraf": "^2.6.1",
     "tslint": "^5.0.0",
-    "typescript": "^2.2.1",
+    "typescript": "^3.6.4",
```

Comment:
A Typescript major version bump is required to compile the opentelemetry types. I don't think this is a breaking change since TS3 types are only used on the dev dependency side... But it may be a reason to implement our own tracer instead of using one from OT.